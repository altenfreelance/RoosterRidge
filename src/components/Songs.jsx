import React from 'react';
const playerStyle = { backgroundColor: 'rgba(128,0,0, 0.8)' }

function Songs() {
    return (
        <div>
            <h1>Audio</h1>
            {playlist.map((song) => {return (
            <div key={song.title} className="my-2 mx-5 rounded p-4" style={playerStyle}>
                <h4 style={{ color: '' }}>{song.title}</h4>
                <audio controls src={song.url} style={{ width: "100%" }} />
            </div>);
            })}
        </div>
    );
}

const playlist = [
    {
        url: 'audio/BloodInTheSoil.mp3',
        title: 'Blood In the Soil',
    },
    {
        url: 'audio/BloodInTheSoil.mp3',
        title: '2 In the Soil',
    }
]

export default Songs