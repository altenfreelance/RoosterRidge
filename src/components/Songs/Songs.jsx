import React from 'react';

function Songs() {
    return (
        <div>
            <h1>New Music Coming Soon!</h1>
            <p>We are releasing an album late spring of 2022. Stay up to date with us on <a href="https://www.facebook.com/RoosterRidgeOfficial/"> Facebook </a> for updates and promotions!</p>
            <br/>
        </div>
    );
}


export default Songs