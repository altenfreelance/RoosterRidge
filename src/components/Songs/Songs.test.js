import React from 'react';
import Songs from './Songs';

import {render, cleanup} from '@testing-library/react';
import '@testing-library/jest-dom';

afterEach(cleanup);

it('renders (matches snapshot) ', () => {
    const {asFragment} = render(<Songs />);
    expect(asFragment()).toMatchSnapshot();
});
