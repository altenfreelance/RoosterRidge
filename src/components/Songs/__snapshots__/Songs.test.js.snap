// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders (matches snapshot)  1`] = `
<DocumentFragment>
  <div>
    <h1>
      Audio
    </h1>
    <div
      class="my-2 mx-4 p-4"
      style="background-color: rgba(128, 0, 0, 0.8); border-radius: 30px;"
    >
      <h4>
        Blood In the Soil (12/20/19)
      </h4>
      <audio
        controls=""
        src="audio/BloodInTheSoil.mp3"
        style="width: 100%;"
      />
    </div>
    <div
      class="my-2 mx-4 p-4"
      style="background-color: rgba(128, 0, 0, 0.8); border-radius: 30px;"
    >
      <h4>
        Fools Gold (12/31/19)
      </h4>
      <audio
        controls=""
        src="audio/FoolsGold.mp3"
        style="width: 100%;"
      />
    </div>
    <div
      class="my-2 mx-4 p-4"
      style="background-color: rgba(128, 0, 0, 0.8); border-radius: 30px;"
    >
      <h4>
        Crawford County Critter Line (12/31/19)
      </h4>
      <audio
        controls=""
        src="audio/CCCL.mp3"
        style="width: 100%;"
      />
    </div>
    <div
      class="my-2 mx-4 p-4"
      style="background-color: rgba(128, 0, 0, 0.8); border-radius: 30px;"
    >
      <h4>
        Atlantic City (12/31/19)
      </h4>
      <audio
        controls=""
        src="audio/AtlanticCity.mp3"
        style="width: 100%;"
      />
    </div>
    <div
      class="my-2 mx-4 p-4"
      style="background-color: rgba(128, 0, 0, 0.8); border-radius: 30px;"
    >
      <h4>
        Cocaine Habit (12/31/19)
      </h4>
      <audio
        controls=""
        src="audio/CocaineHabit.mp3"
        style="width: 100%;"
      />
    </div>
  </div>
</DocumentFragment>
`;
